### DISCUSSION TITLE
Mayor System

### DISCUSSION THREAD

**Discussion by TFury30** [EveronLife/EveronLife#217] [Ideas]
Hello,

How about a Mayor system where people can elect a mayor at the Town hall. 
The Mayor will be able to Increase the Taxes from 10% (normal taxes)  for fuel and other stuff. To up to 20%.
The extra money earned will be sent partially to the mayor's paycheck. 

If a mayor is elected, he will get access to the mayor's hall. Where he can change the taxes from for the duration until the next election (every  Hour).

**Comment by Ezcoo** [comment]
I think it's a great initiative!

Power conflicts create often the most interesting RP scenarios and a powerful mayor character could be a great way to achieve that ingame.

Mayor could also have an option to change the laws (that wouldn't override server rules) ingame and have a wide range, say, from e.g. 0 to 100 % to change rates of different taxes, like vehicles, fuel, licenses, guns, ammo and food. Everything that the taxes would generate would go to mayor's bank account. They could then use the money like they wish by e.g. helping poor citizens or using it to fund their possible secret drug cartel!

Mayor could also activate martial law instead of the cop chief. It would give the mayor more power and create interesting RP scenarios. Martial law could allow cops to carry and use military equipment, like automatic guns, snipers and armored vehicles.

In addition to those, I think that the mayor should possibly be elected with only votes given in person at town hall(s). It would create a motive for the illegal players too to show up at the major towns (and take the risk of getting caught or chased) to prevent them from just lurking nowhere to avoid the cops.

Thumbs up for the idea from here!
